### ISSUE TITLE
nicht erreichbar unter www.shop.meilenwoelfe.de

### ISSUE THREAD

**Issue by sinnloserName4** [lehoffma/memo#190]
dort fehlt wohl ein dns eintrag

**Comment by lehoffma** [comment]
Yep. WWW is auch nur ne subdomain und bei uns is die subdomain halt shop :D Finds aber ohne www eigentlich sogar schicker .

**Comment by sinnloserName4** [comment]
möchte halt gerne eine weiterleitung von www auf ohne www haben. Hab dafür schon den ersten supportauftrag bekommen, weil sörens link eben nicht geht.

**Comment by sinnloserName4** [comment]
Hab schon einiges (ohne Erfolg) probiert:

.htaccess datei in das / verzeichnis legen und auf dieses umleiten
www.shop.meilenwoelfe.de als subdomain anlegen

www.* funktioniert nicht mal wenn ich bei strato intern auf deren paket umleite
